ports/kikai/Kikai.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/kikai"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```

Comment:
LD_LIBRRAY_PATH export is not correct.

Reply:
What is the correct command?

Reply:
> What is the correct command?
It should be something like this

`export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"`

With `libs` renamed to `libs.armhf` if we stick to the multiarch logic but I'll let @Cebion confirm that as it's a 32bits only port.

Reply:
> > What is the correct command?
> 
> It should be something like this
> 
> `export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"`
> 
> With `libs` renamed to `libs.armhf` if we stick to the multiarch logic but I'll let @Cebion confirm that as it's a 32bits only port.

Correct, thing is that its only 32 bit and nothing else we didn'Tbother with armhf.
So correct would be just ```export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"```

Reply:
or even just export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"